ui/src/app/utils/zendesk.ts

```diff
@@ -1,4 +1,69 @@
 import {ElementRef} from '@angular/core';
+import {environment} from 'environments/environment';
+
+interface ZendeskUrls {
+  billing: string,
+  communityForum: string,
+  createBillingAccount: string,
+  dataDictionary: string,
+  faq: string,
+  gettingStarted: string,
+  tableOfContents: string,
+  researchPurpose: string,
+}
+
+/**
+ * A set of support URLs for the current environment. These need to be
+ * parameterized per environment since the Zendesk support content IDs are not
+ * stable across the prod and sandbox Zendesk instances.
+ */
+export const supportUrls: ZendeskUrls = ((baseUrl) => {
```

Comment:
nit: What is the benefit of wrapping everything in an IIFE here? The only export is a ZendeskUrls object. Not actually a big difference, but it might be clearer to do:

```
const baseUrl = environment.zendeskHelpCenterUrl;

... most of function body ...

let supportUrls: ZendeskUrls = urls[prodHost];
if (urls[host]) {
  supportUrls = urls[host];
}
export supportUrls;
```

Reply:
Primarily stylistic preference. Generally I prefer to minimize the footprint in the package context. The state and helpers I've created (for the sole purpose of initializing these URLs) are scoped as tightly as possible and don't pollute the top level context of this file - the top-level approach leaves open the possibility of references / closures from any of the functions defined in the package. Of course, we wouldn't create any such references here, but as a first-time reader of this file I'd need to think/search to rule this out.

Visually, I also find it easier as a reader to scan this package to understand what is being exported, and what's just the internal code details. The visual hierarchy is easier to scan than picking out the `export` keyword from several top-level statements.